### PR TITLE
Updates to panel in the PL library and a few SCSS updates

### PR DESCRIPTION
**Pull request by line47**
Moved and renamed some panel related patterns under the panel section of the pattern library.
Updated some panel SCSS to streamline the panel style.

### CODE REVIEW

File: docs/src/components/features/panel/panel--org-expanded.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<article class="panel">
+<section class="panel">
```
Comment (line47) [inline docs/src/components/features/panel/panel--org-expanded.html:1]: Changing `<article>` to `<section>` as it has more semantic meaning for this situation.

File: src/css/components/panel.scss [hunk 2]
```diff
@@ -97,18 +97,7 @@ h4 + .panel {
   align-items: center;
   display: flex;
   justify-content: space-between;
-  margin-top: $grid-4;
```
Comment (line47) [inline src/css/components/panel.scss:100]: Removing top margin on panel headers to bring styling closer to Sketch files.

File: src/css/components/panel.scss [hunk 3]
```diff
@@ -97,18 +97,7 @@ h4 + .panel {
...
-  // TODO remove elements from this rule.
-  h1,
-  h2,
-  h3,
-  h4 {
-    font-size: $sans-s2;
-    font-weight: 700;
-  }
-
 }
```
Comment (line47) [inline src/css/components/panel.scss:101]: Removing H tags here as we are using the class `.panel-title` for defining.

File: docs/src/components/features/panel/panel.html [hunk 4]
```diff
@@ -1,35 +1,37 @@
-<article class="panel">
-  <h1 class="panel-title">Panel title</h1>
-  <div class="panel-rows">
-    <section class="panel-group">
-      <header class="panel-header">
-        <h2>Group header</h2>
-        <span>Secondary information</span>
+<section class="panel">
+    <header class="panel-header row-center-x">
```
Comment (msecret) [inline docs/src/components/features/panel/panel.html:2]: Can we remove `row-center-x`? Doing so could move this out of prototype. Here and elsewhere.

**Review by msecret — APPROVED** [review]

File: docs/src/components/features/panel/panel.html (hunk 4 above)
Comment (line47) [reply docs/src/components/features/panel/panel.html:2]: I'm working on another batch of updates that addresses this. (removes)
I have a bit more work to get through on it.

Comment (msecret) [reply docs/src/components/features/panel/panel.html:2]: This LGTM then

**Pull request merged**